client/src/components/summary/summary.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { ComponentDiv } from '../sharedStyledComponents';
+import TagList from './summaryTags';
+import ZReview from './summaryReview';
+
+export const SummaryTitle = styled.h1`
+  font: 36px/44px 'Calibre-Medium';
+  letter-spacing: .086em;
+  text-transform: uppercase;
+  margin: 0;
+  padding-top: 20px;
+`;
+
+export const SummarySubtitle = styled.h2`
+  font: 15px/24px 'Calibre-Regular';
+  letter-spacing: 0.013em;
+  color: #656666;
+  margin: 0;
+  padding-top: 2px;
+`;
+
+class Summary extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      restaurant: {},
+    };
+  }
+
+  componentDidMount() {
+    const { id } = this.props;
+
+    this.getRestaurant(id);
+  }
+
+  getRestaurant(id) {
+    fetch(`/restaurants/${id}`)
+      .then(res => res.json())
+      .then((data) => {
+        this.setState({
+          restaurant: data,
+        });
+      });
+  }
+
+  render() {
+    const { restaurant } = this.state;
+
+    return (
```

Comment:
No handling for when restaurant is {} ?
wouldn't restaurant.(properties) be undefined at this point?